README.md

```diff
@@ -836,6 +836,28 @@ For a complete replacement with namespace and delegation, refer to https://githu
   } else {
     document.addEventListener('DOMContentLoaded', eventHandler);
   }
+  
+  // Example 2
+  // Async Method - Ternary - No Arrow IE
+  (async function() {
+      (document.readyState !== 'loading') ?
+          console.log('Loading') // EventHandler
+            : document.addEventListener('DOMContentLoaded', 
+            function() {
+                console.log('Loaded') // EventHandler
+            });
+  })();
+  
+  // Example 3
+  // Non Async Method - Ternary - No Arrow IE
+  (function()  {
+  (document.readyState !== 'loading') ?
+    console.log('Loading') // EventHandler
```

Comment:
use EventHandler, not `console.log`

Reply:
Okay, will fix it. Thanks for having patience with me. Learning git.